annet/adapters/netbox/common/query.py

```diff
@@ -5,7 +5,7 @@
 from annet.storage import Query
 
 FIELD_VALUE_SEPARATOR = ":"
-ALLOWED_GLOB_GROUPS = ["site", "tag", "role"]
+ALLOWED_GLOB_GROUPS = ["site", "tag", "role", "device_type"]
```

Comment:
please add to `Filter` model below as well

Reply:
Added